scripts/miro_updates.py

```diff
@@ -461,3 +481,28 @@ def update_miro_image_suppressions_doc():
         print(
             f"*** To approve these changes, visit https://github.com/wellcomecollection/private/pull/{new_pr_number}"
         )
+
+
+def register_on_dlcs(origin_url, miro_id):
+    dlcs_response = dlcs_api_client().post(
+        f"https://api.dlcs.io/customers/2/queue/priority",
+        json={
+            "@type": "Collection",
+            "member": [
+                {
+                    "space": "8",
+                    "origin": origin_url,
+                    "id": miro_id,
+                    "mediaType": "image/jpeg",
+                }
+            ],
+        },
+    )
+    print(dlcs_response.text)
```

Comment:
Is this text useful, or left here for debug purposes?

Reply:
It's useful.  It includes the batch's URL in the DLCS API so you can look at it to know when it has completed, rather than rushing straight over to the website and getting upset that it isn't there.

I'll make that clearer.